frontend/pages/index.vue

```diff
@@ -1,14 +1,275 @@
+          Wahrnehmung auf die Probe gestellt. Ein Dreieck erscheint in der Mitte
+          des Bildschirms und zeigt in eine von vier Richtungen:
+          <b>Norden, Osten, Süden oder Westen</b>. Ihre Aufgabe ist es, so
+          schnell wie möglich die Richtung zu bestimmen und die entsprechende
+          <b>Pfeiltaste</b> zu drücken. Die Schwierigkeit erhöht sich, indem die
+          <b>Größe</b> des Dreiecks variiert und der <b>Kontrast</b> zum
+          Hintergrund verändert wird. Im "Trisight Mode" müssen Sie gegen die
+          Zeit antreten: Ein Timer läuft, und Ihre Punktezahl wird gezählt,
+          basierend auf der Schnelligkeit und Richtigkeit Ihrer Antworten. Beim
+          klassischen Sehtest gibt es keinen Timer, sodass Sie sich in Ruhe auf
+          die Aufgabe konzentrieren können. Testen Sie Ihre Fähigkeiten und
+          finden Sie heraus, wie schnell Ihre Augen wirklich sind!
+        </p>
+      </div>
+      <div class="image-container">
+        <img src="../images/arrow-keys.png" />
+      </div>
+    </div>
+    <div class="info-section">
+      <div class="text-container">
+        <h2>Ansteigende Schwierigkeit</h2>
+        <p>
+          Im Spiel "Trisight Mode" steigt die Schwierigkeit nach und nach an, um
+          Ihre visuelle Wahrnehmung und Reaktionszeit immer weiter zu fordern.
+          Zu Beginn erscheint ein großes, deutlich sichtbares Dreieck auf einem
+          kontrastreichen Hintergrund. Die Richtung – Norden, Osten, Süden oder
+          Westen – lässt sich leicht erkennen, und Sie können schnell die
+          richtige Pfeiltaste drücken.
+          <b
+            >Mit fortschreitendem Spiel wird das Dreieck jedoch kleiner und der
+            Kontrast zum Hintergrund nimmt ab</b
+          >. Es wird zunehmend schwieriger, die Richtung des Dreiecks sofort zu
+          bestimmen, da es sich optisch weniger abhebt und mehr Konzentration
+          erfordert. Zusätzlich läuft im "Trisight Mode" ein Timer, und Ihre
+          Punktezahl hängt von der Schnelligkeit und Richtigkeit Ihrer Antworten
+          ab. Jede Sekunde zählt, während die Herausforderung wächst – je
+          schneller Sie die Richtung erkennen und die richtige Taste drücken,
+          desto höher ist Ihre Punktzahl. Das Spiel fordert Ihre Augen und Ihre
+          Reaktionsfähigkeit auf höchstem Niveau!
+        </p>
+      </div>
+      <div class="image-container">
+        <div class="circle-background">
+          <img class="triangle-image-2" src="../images/blue-neon-triangle.png" alt="Blue Neon Triangle" />
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {};
+  },
+};
+</script>
+
+<style scoped>
+.homepage {
+  width: 100%;
+  overflow: hidden;
+}
+
+/* Obere Sektion */
+.intro-section {
+  background-color: #185262;
+  background-image: url(../images\trisight-background-home-desktop.png);
+  color: white;
+  min-height: 90vh;
```

Comment:
If we do this with 90vh then maybe we should change the height of the header to 10vh because on my screen it does not cover the full screen: 
![image](https://github.com/user-attachments/assets/765dc02b-efa7-4d8f-a99c-87035436bdd3)